eq-author/src/App/introduction/Design/IntroductionEditor/index.js

```diff
@@ -146,7 +145,7 @@ export const IntroductionEditor = ({
       <Section>
         <Padding>
           <SectionTitle style={{ marginBottom: "0" }}>
-            Introduction content
+            Introduction contenttttt
```

Comment:
```suggestion
            Introduction content
```

With this change, the tests will also fail due to being used in a snapshot test - automatically updating snapshots will resolve this